models/dataset.go

```diff
@@ -452,52 +452,43 @@ func (ed *EditionUpdate) PublishLinks(ctx context.Context, host string, versionL
 	return nil
 }
 
-// ValidateDataset checks the dataset has invalid fields
-func ValidateDataset(ctx context.Context, dataset *Dataset) error {
-
-	var invalidFields []string
-	if dataset.URI != "" {
-		dataset.URI = strings.TrimSpace(dataset.URI)
-		_, err := url.Parse(dataset.URI)
-		if err != nil {
-			invalidFields = append(invalidFields, "URI")
-			log.Event(ctx, "error parsing URI", log.ERROR, log.Error(err))
-		}
+// CleanDataset trims any hrefs contained in the database
```

Comment:
The comment is missing mention of URI, so comment should possibly be:

// CleanDataset trims URI and any HRefs contained in the database

Reply:
i see the comment has been updated